atticmatic/backends/borg.py

```diff
@@ -7,7 +7,14 @@
 
 COMMAND = 'borg'
 CONFIG_FORMAT = (
-    shared.CONFIG_FORMAT[0],  # location
+    Section_format(
+        'location',
+        (
+            option('source_directories'),
+            option('one_file_system', required=False),
```

Comment:
It's probably a good idea here to pass in `value_type=bool`. Note that if you do that, you'll have to alter `type_getter` in `parse_section_options()`, adding something like `bool: parser.getboolean,`